Fog.js

```diff
@@ -995,9 +989,7 @@ function redraw_light_walls(clear=true){
 		let adjustedScale = scale/window.CURRENT_SCENE_DATA.scale_factor;
 
 		if (shape == "line" && ($('#wall_button').hasClass('button-enabled') || ($('#fog_button').hasClass('button-enabled') && $('[data-shape="paint-bucket"]').hasClass('button-enabled')))) {
-			canvas = document.getElementById("temp_overlay");
-			ctx = canvas.getContext("2d");
-			drawLine(ctx,x, y, width, height, color, lineWidth, scale);		
+			drawLine(canvas, x, y, width, height, color, lineWidth, scale);		

```

Comment:
Sorry missed this yesterday just noticed while testing the memory stuff - canvas should be ctx in this draw line.

Reply:
🤦‍♂️ yep, nice catch